deployment/provider-azure-installer-windows.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: csi-secrets-store-provider-azure
+  name: csi-secrets-store-provider-azure
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: csi-secrets-store-provider-azure
+  template:
+    metadata:
+      labels:
+        app: csi-secrets-store-provider-azure
+    spec:
+      nodeSelector:
+        beta.kubernetes.io/os: windows
+      containers:
+        - name: provider-azure-installer
+          image: aramase/secrets-store-csi-driver-provider-azure:win00
```

Comment:
pls create an issue to update this after we cut a new release

Reply:
Opened an issue to track updating image - https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/38